app/views/pages/profile.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div class="row d-flex mt-4">
-    <div class="card col-5 justify-content-center align-items-center" style="background-color: #0B0C10;">
+    <div class="card col-5 align-items-center" style="background-color: #0B0C10;">
```

Comment:
Please change the colors according to the correct convention -> $...... in the css stylesheets